doc/final/final_report.tex

```diff
@@ -81,10 +82,97 @@ \section{Introduction}
 \section{Review of related work}
 
 \section{Requirements}
+\subsection{Team work in requirement phase} The requirement analysis is a basic part of the whole project, which will influence every iterative process in the future development. So at the requirement analysis phase, we have twice meetings to discuss and define the requirements. We focused on The Task in Introductory Lecture, and extracted the list of features of the Traffic Simulation System. These features are presented in our initial report clearly.
```

Comment:
I think it's better if we include what we have in the initial report instead of just referring to that. It will make the final report more self-contained and the reader does not need to switch between documents.